website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -0,0 +1,94 @@
+# `invoke`
+
+```python
+def invoke(
+    contract_address: int,
+    function_name: str,
+    inputs: list[int] | dict[str, Any] | None = None,
+    *,
+    config: SignedCheatcodeConfig | None = None,
+):
+```
+
+`SignedCheatcodeConfig` stores configuration used in cheatcodes that can be signed (invoke, deploy).
```

Comment:
```suggestion
```

This line:
- exists in the `Fees` section
- should be below "cheatcode introduction"

Btw. we don't have migrator deploy cheatcode. `deploy_contract` cheatcode doesn't support signing from what I can see.

Reply:
Ah yes, my bad